models/models.js

```diff
@@ -1,12 +1,112 @@
-var mongoose = require("mongoose");
-var userSchema = new mongoose.Schema({
-    username: String,
-    email: String,
-    password: String,
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const bcrypt = require('bcrypt-nodejs');
+
+
+
+let usernameLengthChecker = (username) => {
```

Comment:
if you create functions this way, it should be const, not let
But in general prefer classic approach while creating funcitons